lib/mycha_installer.coffee

```diff
@@ -0,0 +1,67 @@
+async = require 'async'
+{exec} = require 'child_process'
+fsExtra = require 'fs-extra'
+path = require 'path'
+
+
+class MychaInstaller
+
+  constructor: ({@cwd, @testHelperPath}) ->
+
+
+  install: (done) ->
+    async.parallel [
+      @_installDependencies
+      @_writeMychaConfig
+      @_writeTestHelper
+    ], done
+
+
+  _installDependencies: (done) =>
+    exec "npm install --save-dev chai sinon sinon-chai", {@cwd}, done
+
+
+  _writeMychaConfig: (done) =>
```

Comment:
Okay, I forgot that Mycha can be installed locally and added to the dev dependencies with a single command as well. 

The advantage of installing Mycha globally is that it would make adding Mycha to a lot of repos easier, because all you have to do is `mycha install` instead of `npm install --save-dev mycha`